dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -531,6 +531,10 @@ const backgroundSpeechBubble = (format: ArticleFormat): string => {
 	return pillarPalette[format.theme].main;
 };
 
+const backgroundLiveblogMobile = (): string => {
```

Comment:
This feels a bit off. We pass `format` to `decidePalette` and ask it to choose the colour for a context based on the `format` of the article. But in this case we're saying `backgroundLiveblogMobile`. So the fact it is a liveblog is baked into the property instead of `format.Design === ArticleDesign.LiveBlog`, you see?

I'm also wondering if we can't use a media query here? It sounds like we want a particular colour for `mobile`?

Reply:
Yeah, I have to say this is a sort of preliminary offering as I wanted to talk to you about how to do this in keeping with the decidePalette pattern. Would you have time this morning to have a quick chat about it?